platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -167,33 +175,57 @@ public void destroy() throws ApplicationServiceException {
     }
   }
 
+  @Override
+  public void installProfile(String profile, boolean noRefresh) {
+    try {
+      EnumSet<FeaturesService.Option> options = EnumSet.noneOf(FeaturesService.Option.class);
+      if (noRefresh) {
+        options.add(FeaturesService.Option.NoAutoRefreshBundles);
+      }
+      featuresService.installFeature(profile, options);
+    } catch (Exception e) {
+      LOGGER.error("Could not start profile feature: {}", profile, e);
```

Comment:
Is there anything that an admin can do about these errors? If so, we should say as much in each of these log messages so that we are following our logging guidelines.

Reply:
Not really. Admins aren't going to debug OSGi dependency issues. If they're seeing this it's gotta be a bug. I can't think of a situation where this is expected behavior in production. It means we shipped broken stuff.